src/lib/redux/actions/app.ts

```diff
@@ -8,6 +8,11 @@ export const changeTheme = (nameTheme: string) => ({
 	payload: { nameTheme },
 });
 
+export const saveFavoriteRealm = (realm_name: string | undefined) => ({
```

Comment:
camel case ?

Reply:
useless because it's not a component but just a function